root/static/scripts/edit/URLCleanup.js

```diff
@@ -1898,6 +1898,37 @@ const CLEANUPS = {
       return {result: /^https:\/\/myspace\.com\//.test(url)};
     },
   },
+  'napster': {
+    match: [new RegExp('^(https?://)?((app|www|[a-z]{2})\\.)?napster\\.com', 'i')],
+    type: LINK_TYPES.streamingpaid,
+    clean: function (url) {
+      url = url.replace(/^http:\/\//, 'https://');
+      // Standardise on US (host country) for multi-country redirect
+      url = url.replace(/^https:\/\/((app|www)\.)?napster/, 'https://us.napster');
+      url = url.replace(/[#?].*$/, '');
+      return url;
+    },
+    validate: function (url, id) {
+      if (/\/(alb|art|tra)\.[\d]+/i.test(url)) {
+        return {
+          error: l(
+            `This is a redirect link. Please follow the redirect
+             and add the destination link instead.`,
+          ),  
+          result: false,
+        }
+      }
```

Comment:
LGTM but there is no test for it.

Reply:
Should the test just make sure it's disallowed? We don't currently test for error messages at all, although I guess we could change the tests to do so...

Reply:
Yes. Testing error messages would be nice but far beyond the scope of this PR.

Reply:
Also, I would suggest to link directly to the redirect from the error message as follows (not tested):

```suggestion
      if (/\/(alb|art|tra)\.[\d]+/i.test(url)) {
        return {
          error: exp.l(
            `This is a redirect link. Please open {redirect_url|your link}
             and add the destination link instead.`,
            {
              redirect_url: {
                href: url,
                rel: 'noopener noreferrer',
                target: '_blank',
              },
            },
          ),  
          result: false,
        }
      }
```